gulpfile.js

```diff
@@ -340,7 +340,7 @@ var cycloneDxPipe = (flavour, ...ps) => {
       mergeFile.inputFiles = files.map(f => f.path)
       return [mergeFile]
     }))
-    .pipe(exec(file => `cyclonedx merge \
+    .pipe(exec(file => `npm run cyclonedx-run -- merge \
```

Comment:
Running `cyclonedx` was failing on my machine but not the latter as it was using `cyclonedx-node` from installed node's binaries